src/components/rangeslider/draw.js

```diff
@@ -92,8 +95,13 @@ module.exports = function(gd) {
         opts._height = (fullLayout.height - margin.b - margin.t) * opts.thickness;
         opts._offsetShift = Math.floor(opts.borderwidth / 2);
 
-        var x = margin.l + (graphSize.w * domain[0]),
-            y = fullLayout.height - opts._height - margin.b;
+        var x = margin.l + (graphSize.w * domain[0]);
+
+        var y = (
+            margin.t + graphSize.h * (1 - oppDomain[0]) +
+            tickHeight +
+            opts._offsetShift + constants.extraPad
+        );
```

Comment:
I like it, thanks! Both the cleaned up edges and the wider grabbers. The one thing that still strikes me as odd is how the corners of the grabbers sometimes get rounded off - one pixel that you'd have expected to be black is missing sometimes. Seems like this violates the spirit of `crispEdges` but oh well. Fiddling in the inspector it seems like if I offset the grabbers by half a pixel in both x and y the corners get drawn correctly.